include/SharedFirmwareTypes.h

```diff
@@ -361,6 +361,7 @@ struct VCFInterfaceData_s
     SteeringSensorData_s steering_data;
     DashInputState_s dash_input_state; // Direct button signals from the dashboard IOExpander
     CurrentSensorData_s current_sensor_data;
+    EthernetLinkData_s ethernet_link_data;
```

Comment:
There are different "ethernet link" values depending on whether we're on VCF or VCR. Since different things can be "linked", we can't reuse the EthernetLinkData_s struct. We instead need to make a separate struct VCFEthernetLinkData_s with the corresponding booleans. For good measure, you might want to also rename EthernetLinkData_s to VCREthernetLinkData_s.